### PR TITLE
Vue tsc

### PR DESCRIPTION
**Pull request by cawa-93**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "pretypecheck": "npm run buildEnvTypes",
     "typecheck-main": "tsc --noEmit -p packages/main/tsconfig.json",
     "typecheck-preload": "tsc --noEmit -p packages/preload/tsconfig.json",
-    "typecheck-renderer": "tsc --noEmit -p packages/renderer/tsconfig.json",
+    "typecheck-renderer": "cd packages/renderer && vue-tsc --noEmit",
```
Comment (johnsoncodehk) [inline package.json:21]: You could use `vue-tsc --noEmit -p packages/renderer/tsconfig.json`.

File: packages/preload/types/electron-api.d.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 interface ElectronApi {
-  readonly versions: Record<string, string>
+  readonly versions: Readonly<Record<string, string>> & { [Symbol.iterator](): IterableIterator<Record<string, string>> }
```
Comment (johnsoncodehk) [inline packages/preload/types/electron-api.d.ts:2]: `IterableIterator<Record<string, string>>` should change to `IterableIterator<string>`.

File: package.json (hunk 1 above)
Comment (cawa-93) [reply package.json]: Hmm... I've tried this, but it doesn't seem to work.
```
vue-tsc --noEmit -p packages/renderer/tsconfig.json
```
Not founded error. But 
```
cd packages/renderer && vue-tsc --noEmit
```
does.

I just added a [commit with an error](https://github.com/cawa-93/vite-electron-builder/pull/162/commits/e1e4e7daf70209e6413df955baddc686fa9a9a64#diff-fc9f137a66dd3adaed4151ff937426b60f925a59179b4057cf540b8ea9f572abR24). But the type check was [successful](https://github.com/cawa-93/vite-electron-builder/pull/162/checks?check_run_id=2122363220#step:8:6).

Comment (johnsoncodehk) [reply package.json]: Sorry it's a bug and I fixed it: https://github.com/johnsoncodehk/vue-tsc/commit/e0d8a363aa4cce30dca47b8ee540a85190af8365

Not yet publish to 0.0.14, please use old script for now.

**Pull request merged**